src/runtime/client-event-manager.js

```diff
@@ -86,7 +86,7 @@ export class ClientEventManager {
    * @param {!Promise} configuredPromise
    */
   constructor(configuredPromise) {
-    /** @private {!Array<function(!../api/client-event-manager-api.ClientEvent)>} */
+    /** @private {!Array<function(!../api/client-event-manager-api.ClientEvent, Object)>} */
```

Comment:
And also listeners gets optional params here so that we can directly pass `eventParams` in the listeners in ` logEvent` below?